code/game/objects/items/robot/ai_upgrades.dm

```diff
@@ -4,7 +4,7 @@
 //Malf Picker
 /obj/item/malf_upgrade
 	name = "combat software upgrade"
-	desc = "A highly illegal, highly dangerous upgrade for artificial intelligence units, granting them a variety of powers as well as the ability to hack APCs."
+	desc = "A highly illegal, highly dangerous upgrade for artificial intelligence units, granting them a variety of powers as well as the ability to hack APCs.<br> This upgrade does not override any active laws, and must be applied directly to an active AI core."
```

Comment:
Space after `<br>` unnecessary (but should have no ill effects)

Reply:
Oh that's good to know, let me remove the space.